crates/hir_def/src/body/tests/block.rs

```diff
@@ -352,25 +350,18 @@ fn is_visible_from_same_def_map() {
     check_at(
         r#"
 fn outer() {
-    mod command {
-        use crate::name;
-    }
-
     mod tests {
         use super::*;
     }
+    use crate::name;
```

Comment:
Reordered so that the old code path was still hit with the more correct inner item handling.